packages/jobs/lib/execution/webhook.ts

```diff
@@ -161,11 +166,71 @@ export async function handleWebhookSuccess({ nangoProps }: { nangoProps: NangoPr
         internalIntegrationId: nangoProps.syncConfig.nango_config_id
     });
 
-    await updateSyncJobStatus(nangoProps.syncJobId!, SyncStatus.SUCCESS);
+    const syncJob = await updateSyncJobStatus(nangoProps.syncJobId!, SyncStatus.SUCCESS);
+
+    if (!syncJob) {
+        throw new Error(`Failed to update sync job status to SUCCESS for sync job: ${nangoProps.syncJobId}`);
+    }
+
+    const webhookSettings = await externalWebhookService.get(nangoProps.environmentId);
+    const logCtx = await logContextGetter.get({ id: String(nangoProps.activityLogId) });
+    const environment = await environmentService.getById(nangoProps.environmentId);
+    if (environment) {
+        for (const model of nangoProps.syncConfig.models) {
+            const span = tracer.startSpan('jobs.webhook.webhook', {
+                tags: {
+                    environmentId: nangoProps.environmentId,
+                    connectionId: nangoProps.connectionId,
+                    syncId: nangoProps.syncId,
+                    syncJobId: nangoProps.syncJobId,
+                    syncSuccess: true,
+                    model
+                }
+            });
+            void tracer.scope().activate(span, async () => {
+                try {
+                    const res = await sendSyncWebhook({
+                        connection: {
+                            id: nangoProps.nangoConnectionId!,
+                            connection_id: nangoProps.connectionId,
+                            environment_id: nangoProps.environmentId,
+                            provider_config_key: nangoProps.providerConfigKey
+                        },
+                        environment: environment,
+                        webhookSettings,
+                        syncName: nangoProps.syncConfig.sync_name,
+                        model,
+                        now: nangoProps.startedAt,
+                        success: true,
+                        responseResults: syncJob.result?.[model] || { added: 0, updated: 0, deleted: 0 },
+                        operation: 'WEBHOOK',
+                        logCtx
+                    });
+
+                    if (res.isErr()) {
+                        throw new Error(`Failed to send webhook for webhook: ${nangoProps.syncConfig.sync_name}`);
+                    }
+                } catch (err: unknown) {
+                    span?.setTag('error', err);
+                } finally {
+                    span.finish();
+                }
+            });
+        }
+    }
```

Comment:
this looks very much like the logic in `./sync.ts` but I decided it was not worth consolidating the code because there is some small differences. For example webhook execution don't have to deal with full refresh or track_deletes